server/schemas/integration.graphql

```diff
@@ -0,0 +1,56 @@
+enum IntegrationType {
+  Public
+  Private
+}
+
+type Integration implements Node {
+  id: ID!
+  name: String!
+  description: String
+  logoUrl: URL!
+  type: IntegrationType!
+  token: String!
+  developerId: ID!
+  developer: User!
+  webhook: [Webhook!]!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+}
+
+# Inputs
+input CreateIntegrationInput {
+  name: String!
+  description: String
+  logoUrl: URL!
+  type: IntegrationType!
+}
+
+input UpdateIntegrationInput {
+  integrationId: ID!
+  name: String
+  description: String
+  logoUrl: URL
+}
+
+input DeleteIntegrationInput {
+  integrationId: ID!
+}
+
+# Payload
+type IntegrationPayload {
+  integration: Integration!
+}
+
+type DeleteIntegrationPayload {
+  integrationId: ID!
+}
+
+extend type Query {
+  integrations(type: IntegrationType, keyword: String): [IntegrationPayload]!
+}
```

Comment:
```suggestion
```

I think it is not needed. Developers themselves add integrations directly to the workspace they belong to. No search function is required.

Reply:
we can add the integrations to the `Me` query but how about, the other users want to list the integration to install them?

Reply:
No need. Only private integrations are available at the beginning. Public integrations will not be implemented at least this year.